web/src/components/PoiFilters.tsx

```diff
@@ -60,8 +62,17 @@ const StyledButton = styled(TextButton)`
 `
 
 const StyledIcon = styled(Icon)`
-  width: 24px;
-  height: 24px;
+  width: 30px;
+  height: 30px;
+`
+
+const StyledCheckboxContainer = styled.div`
+  
+    @media (width >= 768px){
+        input[type="checkbox"] {
+            transform: scale(1.3); 
+        }
+    }
 `
```

Comment:
:wrench: Instead of wrapping the checkbox in an extra div, you could just pass the `className` prop to `StyledCheckbox` in [Checkbox.tsx](https://github.com/digitalfabrik/integreat-app/blob/8525cfc1526ffad4c0c381cb2ca43be6073c9437/web/src/components/base/Checkbox.tsx#L33), for more information see the [styled-components docs](https://styled-components.com/docs/basics#styling-any-component).

See an example here: https://github.com/digitalfabrik/integreat-app/blob/main/web/src/components/LastUpdateInfo.tsx

Then you can just do:
```suggestion
const StyledCheckbox = styled(Checkbox)`
  min-width: 20px;
`
```